packages/side-nav/src/CollapseButton/collapse-button.scss

```diff
@@ -0,0 +1,36 @@
+@import "~@hig/styles/mixins/base.scss";
+@import "~@hig/styles/mixins/colors.scss";
+
+.hig__side-nav__module__collapse-button {
+  &:hover {
+    cursor: pointer;
+  }
+
+  .hig__icon {
+    @include vendor-prefix("transform", "rotate(0deg)");
```

Comment:
Can this be applied to the entire component rather than targeting the Icon within?

Reply:
Yeah, we shouldn't break encapsulation. We should either pass in a prop to do accomplish this, or preferably animate a wrapper around the entire component.